Users/coafile.rst

```diff
@@ -39,14 +39,9 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
-It can be used to define the type of files you usually
-don't want to lint like minified files
-and backup files (e.g. ``*.min.js``, ``*.orig``)
-such as the file present in ``.coafile`` in the main
-coala repo.
-
-``default_coafile`` is present in ``coalib``,
-inside the main ``coala`` repo.
+It can be used to define the type of files you usually don't want to lint
+like minified files (e.g. ``*.min.js``) and backup files (e.g. ``*.orig``)
+one such example is there in ``.coafile`` in the main coala repository.
```

Comment:
Also: Please hardcode the example here, our coala file is quite large and complicated, nobody wants to look on that just to get a simple idea.